.github/workflows/docs.yml

```diff
@@ -0,0 +1,31 @@
+name: mobile_docs
+
+on:
+  push:
+    branches:
+    - main
+  pull_request:
+
+jobs:
+  docs:
+    runs-on: ubuntu-20.04
+    timeout-minutes: 20
+    container:
+      image: envoyproxy/envoy-build-ubuntu:b0ff77ae3f25b0bf595f9b8bba46b489723ab446
+    steps:
+    - uses: actions/checkout@v1
```

Comment:
Doesn't `checkout@v3` work? Too old git in the image?

Reply:
`checkout@v3` works.